app/lib/frontend/handlers/pubapi.dart

```diff
@@ -165,6 +165,18 @@ class PubApi {
   ) async =>
       await packageBackend.removeUploader(package, email);
 
+  /// Remove an existing uploader.
+  ///
+  /// This is a different API endpoint than the pub CLI tool is using, because
+  /// we want to use a separate authentication mechanism going forward.
```

Comment:
Why does that mean we need a new end-point?

Reply:
I first thought that the authentication migration will change the credentials that will hit the endpoint (sessionId cookie + CSRF token), but now I've realized that we'll need to support both at the same time, so this doesn't necessarily need a new endpoint.

However, deprecating the pub tool endpoint (#5566) requires a new endpoint for the UI, and moving the email to the payload is more similar to the invite-uploader method.

Reply:
Okay, I'm down with that...